src/main/target/common_post.h

```diff
@@ -58,6 +58,16 @@
 #endif
 #endif
 
+#if !defined(USE_MAG)
```

Comment:
For maintainability's sake I'd move this to right above where the rest of the MAG related defines are.

Reply:
the issue is that these defines are spread across platform, unified and legacy targets and some might define a few hardware variants while others deactivate completely the feature. the other issue is that some of these feature implementations check only for a hardware variant defines ignoring if the main feature is enabled at all causing compilation failures because of missing symbols. dealing with these on `common_post.h` seems to be the most reliable place, and it's actually being used in some other similar situations like `USE_SERIAL_RX` or `USE_TELEMETRY`.

Reply:
Sorry, I wasn’t being clear. I meant you should move them to a better place _inside common_post.h_.